### PR TITLE
add SKIP_RUNTIME and RUNTIME_ONLY in setup

### PR DESCRIPTION
**Pull request by xin3he**
## Type of Change

feature

## Description

add SKIP_RUNTIME when running setup.py
change BACKENDS_ONLY to RUNTIME_ONLY

## Expected Behavior & Potential Risk

SKIP_RUNTIME=True python setup.py install/develop works well without cmake

## How has this PR been tested?

local test

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -254,16 +262,15 @@ def check_submodules():
         keywords='quantization, auto-tuning, post-training static quantization, post-training dynamic quantization, quantization-aware training, tuning strategy',
         license='Apache 2.0',
         url="https://github.com/intel/intel-extension-for-transformers",
-        ext_modules=[CMakeExtension(
-            "intel_extension_for_transformers.neural_engine_py", 'intel_extension_for_transformers/backends/neural_engine/')],
         packages=packages_list,
```
Comment (a32543254) [inline setup.py:265]: you may also need change packages_list, otherwise you will also package neural engine compile python api to env

Comment (xin3he) [reply setup.py:265]: thank you, Bo~

**Comment by hshen14** [comment]
What's the purpose for this PR? Why need env variables?

**Comment by xin3he** [comment]
> What's the purpose for this PR? Why need env variables?

The purpose is to skip building cpp files when using optimization, which takes a long time when install ITREX.

**Review by a32543254 — APPROVED** [review]
LGTM

**Review by hshen14 — APPROVED** [review]

**Pull request merged**